PocketKit/Sources/Sync/Log.swift

```diff
@@ -137,17 +137,17 @@ public class Log {
     public class func log( _ object: Any, level: Level = .info, filename: String = #file, line: Int = #line, column: Int = #column, funcName: String = #function) {
         switch level {
         case .verbose:
-            Log.verbose(object)
+            Log.verbose(object, filename: filename, line: line, column: column, funcName: funcName)
```

Comment:
Missed this in the original logger pr. This passes through the original caller of the top level function to the log statements.